src/components/styles/List.js

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export const ListWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+    color: #667080;
+    max-width: 60%;
+    margin: 2rem auto 0 auto;
+`;
```

Comment:
You have several styled.div in your code, you could just declare one styled.div and work with props and $variant.